frontend/src/constants/valuableNumbers.ts

```diff
@@ -1,2 +1,5 @@
 export const MILLISECONDS_IN_SECOND = 1000
-export const TEN_SECONDS_BEFORE_TOKEN_EXPIRE = 10000
\ No newline at end of file
+export const TEN_SECONDS_BEFORE_TOKEN_EXPIRE = 10000
+
+export const MINUTES_QTY_TWO_CHARS_LONG = 2
+export const SIXTY_MINUTES_IN_HOUR = 60
```

Comment:
`SIXTY_MINUTES_IN_HOUR` => `MINUTES_IN_HOUR`
`MINUTES_QTY_TWO_CHARS_LONG` => `MINUTES_DIGITS_LEN`
regarding LEN instead of QTY - i think it will be more clear, since the expression `string length` is more used rather than `quantity of chars in string`